packages/wrangler/src/bundle.ts

```diff
@@ -13,6 +13,13 @@ import type { DurableObjectBindings } from "./config/environment";
 import type { WorkerRegistry } from "./dev-registry";
 import type { Entry } from "./entry";
 import type { CfModule } from "./worker";
+
+export const CommonESBuildOptions = {
```

Comment:
Do we have a convention around title casing variables (even if they are const)?
Generally in Wrangler we have used upper snake case for constants and reserved title case for types (including classes and enums).